integration/combination/test_function_with_mq.py

```diff
@@ -25,6 +25,7 @@ def companion_stack_outputs(self, get_companion_stack_outputs):
             ),
         ]
     )
+    @nonblocking
```

Comment:
Isn't it missing the `combination/function_with_mq` transform test?

Reply:
The transform test was already added before. https://github.com/aws/serverless-application-model/pull/3045/files